kornia/color/ycbcr.py

```diff
@@ -0,0 +1,116 @@
+import torch
+import torch.nn as nn
+
+
+class RgbToYcbcr(torch.nn.Module):
+    r"""Convert image from RGB to YCbCr.
+    The image data is assumed to be in the range of (0, 1).
+
+    args:
+        image (torch.Tensor): RGB image to be converted to YCbCr
+
+    returns:
+        torch.Tensor: YCbCr version of the image.
+
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 3, H, W)`
+
+    """
+
+    def __init__(self) -> None:
+        super(RgbToYcbcr, self).__init__()
+
+    def forward(self, image: torch.Tensor) -> torch.Tensor:  # type: ignore
+        return rgb_to_ycbcr(image)
+
+
+def rgb_to_ycbcr(image: torch.Tensor) -> torch.Tensor:
+    r"""Convert a RGB image to YCbCr.
+    See :class:`~kornia.color.RgbToYcbcr` for details.
+
+    Args:
+        input (torch.Tensor): RGB Image to be converted to YCbCr.
+
+    Returns:
+        torch.Tensor: YCbCr version of the image.
+    """
+
+    if not torch.is_tensor(image):
+        raise TypeError("Input type is not a torch.Tensor. Got {}".format(
+            type(image)))
+
+    if len(image.shape) < 3 or image.shape[-3] != 3:
+        raise ValueError("Input size must have a shape of (*, 3, H, W).Got {}"
+                         .format(image.shape))
+
+    r: torch.Tensor = image[..., 0, :, :]
+    g: torch.Tensor = image[..., 1, :, :]
+    b: torch.Tensor = image[..., 2, :, :]
+
```

Comment:
I would reference somehow from where did you get those weights. As a first search I found wiki page: https://es.wikipedia.org/wiki/YCbCr

Reply:
Google returns https://www.wdic.org/w/WDIC/YCbCr on the top in Japanese, which provides wrong info.